auth-server/knexfile.js

```diff
@@ -6,16 +6,27 @@ dotenv.load()
 const databaseName = 'governance-auth'
 const connectionUrl = process.env.DATABASE_URL || `postgres://postgres:postgres@localhost:5431/${databaseName}`
 
-console.log('applying migration on ', connectionUrl)
-
-module.exports = {
-	client: 'postgresql',
-	connection: connectionUrl,
-	migrations: {
-		directory: `${__dirname}/migrations`
+const config = {
+	test: {
+		client: 'postgresql',
+		connection: 'postgres://postgres:postgres@localhost:5431/governance-auth-test',
```

Comment:
it'd be nicer to have all connection urls with their secrets together in an .env variable. So have above a:
```js
const testConnectionUrl = process.env.TEST_DATABASE_URL
const devConnectionUrl = process.env.DEV_DATABASE_URL
```

Reply:
done